box/models/ebics_user.rb

```diff
@@ -83,6 +83,19 @@ def activate!
       false
     end
 
+    def refresh_bank_keys!
+      Box.logger.info("Refresh bank keys for account #{id}")
+      client.HPB
+
+      self.encryption_keys = client.send(:dump_keys)
+      save
+      true
```

Comment:
`save` should return true or false. It won't raise an error if it fails. In case you need an error to land in the rescue block below, use `save!`

Reply:
Does save! exist for Sequel. It is not Rails and Active record. I copied the lines from other locations in the same file.

Reply:
oups! I thought this was rails 💨